torch/csrc/jit/mobile/import.cpp

```diff
@@ -303,6 +304,14 @@ void BytecodeDeserializer::parseFunctionSchema(
   }
 }
 
+void BytecodeDeserializer::init_upgrader(mobile::Function* function) {
+  for (auto const& [upgrader_name, upgrader_bytecode] : kUpgraderBytecode) {
+    std::unique_ptr<mobile::Function> func = mobile::Function::get(
```

Comment:
`func` will be released immediately at the end of the loop body, consider storing these functions in a static local container.

Reply:
Yeah resolved by using static